### PR TITLE
Add Rust with Prusto as client

### PR DESCRIPTION
**Pull request by mosabua**
Fyi @nooberfsh .. also we would love to have you as guest on the Trino Community Broadcast to chat more about Trino and Rust. 

Props to @sbernauer for telling me about the client

Preview at https://deploy-preview-589--trino.netlify.app/ecosystem/client#rust

### CODE REVIEW

File: _data/tools.yml [hunk 1]
```diff
@@ -1166,6 +1166,21 @@
       url: https://www.ruby-lang.org/en/
     - urltext: trino-client-ruby
       url: https://github.com/treasure-data/trino-client-ruby
+- name: Rust
+  anchor: rust
+  category: client
+  logo: /assets/images/logos/rust.png
+  logosmall: /assets/images/logos/rust-small.png
+  description: |
+    Rust is a programming language empowering everyone to build reliable and
```
Comment (mosabua) [inline _data/tools.yml:1175]: First sentence is basically headline from the Rust website...

**Review by sbernauer — APPROVED** [review]
Thanks!
FYI, there is also https://github.com/joshthoward/trino-rust-client, but prusto has more functionality and us used by more people

**Comment by mosabua** [comment]
> Thanks! FYI, there is also https://github.com/joshthoward/trino-rust-client, but prusto has more functionality and is used by more people

Yes... @joshthoward had this as a hackday project I think .. maybe he can link to Prusto from his project. Or should we also link to it on the website?

**Review by findepi — APPROVED** [review]

**Comment by sbernauer** [comment]
> Or should we also link to it on the website?

I personally think the current state should be fine, but he can comment in case his repo should  be mentioned as well

**Pull request merged**